esx_service/vmdk_ops.py

```diff
@@ -606,6 +606,13 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
 
     elif cmd == "remove":
         response = removeVMDK(vmdk_path)
+        # remove succeed, remove infomation of this volume from volumes table
```

Comment:
this is the wrong place to manipulate the config DB. It should be done within removeVMDK